functions

```diff
@@ -569,7 +569,7 @@ function install_venv {
     fi
     if [[ ! -f "${venv_path}/bin/activate" ]]; then
         echo "Creating virtual environment in ${venv_path}"
-        virtualenv -p python3 "${venv_path}"
+        python3 -m venv "${venv_path}"
```

Comment:
Will it break older releases still using virtualenv?

Reply:
it shouldn't (not 100% sure though). We've seen opposite issue - where using `-m venv` inside `virtualenv` created venv - was causing issues with recent changes upstream for 3.6 python based distros.

Reply:
furthermore we've got `-m venv` in beokay for some time now, and there was no issue with it